docs/index.md

```diff
@@ -1,17 +1,109 @@
 ---
-title: Tilt API Reference
+title: Tilt API Server Reference
 layout: api
 ---
 
```

Comment:
What does the Tilt API server do? What usecases does it address? Which problems does it solve?

Reply:
There's a whole section on this?

```
## For Dev Environment Maintainers

This reference site is for users who have already set up Tilt, and are:

- trying to diagnose why Tilt isn't behaving like you expect, or

- writing new templates for dev environments to share across teams, or

- scraping metrics about your dev environment to track internally, or

- contributing to Tilt itself (!)
```

Reply:
I see.
I got confused, because that section is introduced as "Here are some things that this guide might help with:", so I understood this to mean, this describes the value of this guide, not the Tilt API itself.
Also, this guide starts with who should not be reading it. I think it would be better to start off with why you *should* read on.
I would suggest a one or two sentence top level explanation of the Tilt API before jumping in with the content.
For example:
``` 
The Tilt Server API allows you to drill down into the depths of Tilt, to get at exactly what you need without all the window-dressing. Consume the API to automate setup, scrape metrics and much more.
```

Reply:
done